crates/apollo-compiler/test_data/diagnostics/0016_schema_with_built_in_scalar_definition.txt

```diff
@@ -1,28 +1,58 @@
 [
     ApolloDiagnostic {
         cache: {
-            15: "0016_schema_with_built_in_scalar_definition.graphql",
+            30: "src/built_in_types.graphql",
+            31: "0016_schema_with_built_in_scalar_definition.graphql",
         },
         location: DiagnosticLocation {
             file_id: FileId {
-                id: 15,
+                id: 31,
             },
-            offset: 30,
-            length: 10,
+            offset: 37,
+            length: 3,
         },
         labels: [
             Label {
                 location: DiagnosticLocation {
                     file_id: FileId {
-                        id: 15,
+                        id: 30,
                     },
-                    offset: 30,
-                    length: 10,
+                    offset: 2902,
+                    length: 3,
                 },
-                text: "remove this scalar definition",
+                text: "previous definition of `Int` here",
+            },
+            Label {
+                location: DiagnosticLocation {
+                    file_id: FileId {
+                        id: 31,
+                    },
+                    offset: 37,
+                    length: 3,
+                },
+                text: "`Int` redefined here",
```

Comment:
this change is actually incorrect and i still need to fix. because built in definitions are now loaded via a source, we exclude any duplicate scalar definitions simply by nature of index map. with redefined built-in scalars, the diagnostic should indicate to remove the redefined scalar.

Reply:
🙏 fixed